OntologyAPI.spec

```diff
@@ -155,6 +155,15 @@ module OntologyAPI {
         int  offset;
     } GetTermsFromWSFeatureParams;
 
+    typedef structure {
+        string name;
+        ID ancestor_term;
+        int ts;
+        string ns;
+        int  limit;
+        int  offset;
+    } GetTermByNameParams;
```

Comment:
some documentation about what each variable should be would be helpful here

Reply:
documentation added